CapstoneProject/ViewControllers/LoginViewController.h

```diff
@@ -0,0 +1,21 @@
+//
+//  LoginViewController.h
+//  CapstoneProject
+//
+//  Created by Apoorva Chilukuri on 7/11/22.
+//
+
+#import <UIKit/UIKit.h>
+#import <Parse/Parse.h>
+//#import <Parse/PFFacebookUtils.h>
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LoginViewController : UIViewController
+@property (weak, nonatomic) IBOutlet UITextView *usernameField;
```

Comment:
Try to move those UI properties inside implementation file (.m) to make them private